### PR TITLE
Fixes and improvements to Announcements/SSOT

### PR DESCRIPTION
**Pull request by fmgoncalves**
Fix a signing bug when there is a line feed char in content. This bug is due to a mismatch between toHex functions on web3py and web3js.

Minor improvements around usability (no login required for announcements page, focus on content, etc)

### CODE REVIEW

File: web/ssot/templates/announcements.html [hunk 1]
```diff
@@ -3,7 +3,17 @@
 
 {% block content %}
 
+{% comment %}
+Ugly TEMPORARY hack follows
```
Comment (dpnova) [inline web/ssot/templates/announcements.html]: I can wear comments like this but I'd rather it describe when it needs to be revisited...

**Comment by dpnova** [comment]
I'm thinking we should render out the addresses and ticks in a list? They are getting split oddly atm... what do you reckon @ejjick  ?

**Comment by ejjick** [comment]
Just tested it. Works, but new lines become visible `<br />`, e.g "Ipsum?<br />Lorem" (we should allow HTML perhaps)

File: web/ssot/templates/announcements.html [hunk 2]
```diff
@@ -3,7 +3,17 @@
...
+{% endcomment %}
+<style>
```
Comment (ejjick) [inline web/ssot/templates/announcements.html]: Inline CSS is a no-no in my book. I can change our code to hide the menu when not logged in though.

**Comment by ejjick** [comment]
> I'm thinking we should render out the addresses and ticks in a list? They are getting split oddly atm... what do you reckon @ejjick ?

I agree @dpnova Got a link to an announcement with 2+ signatures that I can preview to fix the design?

**Comment by dpnova** [comment]
Feel free to pull and make the changes @ejjick

**Comment by ejjick** [comment]
Just pushed some updates.

Let's discuss riot / code style / prototyping vs prod at some point... I agree with having the `ssot-msg.tag` template (it's generated many times and lots of JS going on). I don't agree with having `ssot-header.tag` (only generated once, nothing happening client-side from what I can tell).

**Comment by dpnova** [comment]
I'm going to veto this and say we keep it. :P

**Comment by dpnova** [comment]
ah the header tag? yeah whatever... keep the body one though!

**Comment by dpnova** [comment]
we got bigger balls to grab atm anyway

**Comment by dpnova** [comment]
Also hello world - welcome to our github :)

**Comment by dpnova** [comment]
No breaking db changes here so I'm accepting.

**Pull request merged**